### ISSUE TITLE
Add status "scheduled" / Ready to Run  or  respect the Election "Start Date" 

### ISSUE THREAD

**Issue by masiarek** [Equal-Vote/star-server#497]
As-is:
<img width="784" alt="image" src="https://github.com/Equal-Vote/star-server/assets/857777/30af9723-e12e-4117-b6d8-5fc219a7f4b1">

Should-be:
Allow "scheduling" election - instead of "Voting begins immediately"
- allow choosing the date/time when email notices are sent (it is "show time" or it is "get ready for voting" - voting begins at...)
or
- respect the election Start Date

Until Start date is reached - election should have status:
- "scheduled" / Ready to Run - if all checks are successful
- Incomplete / draft - if some checks failed (e.g. missing emails for some voters, payment not received)

<img width="680" alt="image" src="https://github.com/Equal-Vote/star-server/assets/857777/c29a215d-d1e2-4ca3-ba71-6dd81b9c0063">

Election status: https://docs.google.com/document/d/1kdXQDUkyl6CwABp7bExzjg2L0KgiaaXs3ikeCox54eg/edit

**Comment by ArendPeter** [comment]
Agreed, we should update the wording and functionality to respect the start date

**Comment by ArendPeter** [comment]
The "voting begins immediately" wording is already dynamic, and it will change to "voting begins after your specified start time" if there's a start time. 

That said, you make a good point that the election state needs to be surfaced more clearly. I'll move this to 2.0 and also link it to https://github.com/Equal-Vote/star-server/issues/317
